lib/secretariat/validator.rb

```diff
@@ -58,7 +58,7 @@ def validate_against_schematron
         File.write(docpath, doc, mode: 'wb')
         jarpath = File.join(__dir__, '../..', 'bin', 'schxslt-cli.jar')
         out = `java -jar #{jarpath} -v -d #{docpath} -s #{schematron_path}`
-        return [] if out.match("[valid]")
+        return [] if out.start_with?("[valid]")
```

Comment:
someone suggested to use out.strip, but from my testing it doesn't seem to be needed and "[valid]" is always the immediate start if it is indeed valid.